fast64_internal/sm64/sm64_constants.py

```diff
@@ -1799,6 +2035,29 @@ def __init__(self, geoAddr, level, switchDict):
     ("Custom", "Custom", "Custom"),
 ]
 
+# groups you can use for the combined object export
+groups_obj_export = [
+    ("common0", "common0", "chuckya, boxes, blue coin switch"),
+    ("group0", "group0", "Mario, particles etc."),
+    ("group1", "group1", "Ground objects (Thwomp, Heave-Ho, Hoot etc.)"),
+    ("group2", "group2", "Bully/Blargg"),
+    ("group3", "group3", "King Bob-Omb"),
+    ("group4", "group4", "Water Objects (Unagi, Manta, Clam)"),
+    ("group5", "group5", "Sand Objects (Pokey, Eyerock, klepto)"),
+    ("group6", "group6", "TTM Objects (Monty Mole, uUkiki, Fwoosh)"),
+    ("group7", "group7", "Snow Objects (Mr Blizzard, Spindrift etc.)"),
+    ("group8", "group8", "Cap Switch"),
+    ("group9", "group9", "Haunted Objects (Boo, Mad Piano etc.)"),
+    ("group10", "group10", "Peach/Yoshi"),
+    ("group11", "group11", "THI Ojbects (Lakitu, Wiggler, Bubba)"),
+    ("group12", "group12", "Bowser/Bowser Bomb"),
+    ("group13", "group13", "Water Objects (Skeeter, Treasure Chest etc.)"),
+    ("group14", "group14", "Ground Objects (Piranha Plant, Chain Chomp etc.)"),
+    ("group15", "group15", "Castle Objects (MIPS, Toad etc.)"),
+    ("group16", "group16", "Ice Objects (Chill Bully, Moneybags)"),
+    ("group17", "group17", "Cave Objects (Swoop, Scuttlebug, Dorrie etc.)"),
+    ("Custom", "Custom", "Custom"),
+]
```

Comment:
I still think it would be nice to have the description include the levels in which these are loaded. 
For example, while group 4 includes water objects, it does not get loaded in wdw.
Also, group7 and group 16 get loaded in the exact same two levels, you should probably only include group 7